stable/distribution/README.md

```diff
@@ -0,0 +1,207 @@
+
+* Deploy Mongodb database.
+* Deploy a Redis.
+* Deploy a distributor.
+* Deploy a distribution.
+
+## Requirements
+- A running Kubernetes cluster
+- Dynamic storage provisioning enabled
+- Default StorageClass set to allow services using the default StorageClass for persistent storage
+- A running Artifactory Enterprise Plus
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and setup to use the cluster
+- [Helm](https://helm.sh/) installed and setup to use the cluster (helm init)
+
+## Installing the Chart
+To install the chart with the release name `distribution`:
+```
+helm install --name distribution stable/distribution
+```
+
+### Accessing Distribution
+**NOTE:** It might take a few minutes for Distribution's public IP to become available, and the nodes to complete initial setup.
+Follow the instructions outputted by the install command to get the Distribution IP and URL to access it.
+
+### Updating Distribution
+Once you have a new chart version, you can update your deployment with
+```
+helm upgrade distribution stable/distribution
+```
+
+### Create a unique Master Key
+JFrog Distribution requires a unique master key to be used by all micro-services in the same cluster. By default the chart has one set in values.yaml (`distribution.masterKey`).
+
+**This key is for demo purpose and should not be used in a production environment!**
+
+You should generate a unique one and pass it to the template at install/upgrade time.
+```bash
+# Create a key
+$ export MASTER_KEY=$(openssl rand -hex 32)
+$ echo ${MASTER_KEY}
+
+# Pass the created master key to helm
+$ helm install --set distribution.masterKey=${MASTER_KEY} -n distribution stable/distribution
+```
+**NOTE:** Make sure to pass the same master key with `--set distribution.masterKey=${MASTER_KEY}` on all future calls to `helm install` and `helm upgrade`!
+
+
+### External Databases
+There is an option to use external database services (MongoDB or PostgreSQL) for your Distribution.
+
+#### MongoDB
+To use an external **MongoDB**, You need to set Distribution **MongoDB** connection URL.
+
+For this, pass the parameter: `mongodb.enabled=false,global.mongoUrl=${DISTRIBUTION_MONGODB_CONN_URL},global.mongoAuditUrl=${DISTRIBUTION_MONGODB_AUDIT_URL}`.
+
+**IMPORTANT:** Make sure the DB is already created before deploying Distribution services
+```bash
+# Passing a custom MongoDB to Distribution
+
+# Example
+# MongoDB host: custom-mongodb.local
+# MongoDB port: 27017
+# MongoDB user: distribution
+# MongoDB password: password1_X
+
+$ export DISTRIBUTION_MONGODB_CONN_URL='mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@custom-mongodb.local:27017/${MONGODB_DATABSE}'
+$ export DISTRIBUTION_MONGODB_AUDIT_URL='mongodb://${MONGODB_USER}:${MONGODB_PASSWORD}@custom-mongodb.local:27017/audit?maxpoolsize=500'
+$ helm install -n distribution --set global.mongoUrl=${DISTRIBUTION_MONGODB_CONN_URL},global.mongoAuditUrl=${DISTRIBUTION_MONGODB_AUDIT_URL} incubator/distribution
```

Comment:
This line references incubator but the PR has the chart in stable. Might need to update that.

Reply:
@mattfarina Should I create new PR to deprecate incubator chart?

Reply:
@mattfarina Here is the new PR for deprecating incubator chart https://github.com/helm/charts/pull/7155

Reply:
@davidkarlsen Done